pallets/rc-migrator/src/lib.rs

```diff
@@ -196,6 +208,12 @@ pub mod pallet {
 		type AhWeightInfo: AhWeightInfo;
 		/// The existential deposit on the Asset Hub.
 		type AhExistentialDeposit: Get<<Self as pallet_balances::Config>::Balance>;
+		/// Contains all calls that are allowed during the migration.
+		///
+		/// The calls in here will be available again after the migration.
+		type RcCallEnabledDuringMigration: Contains<<Self as frame_system::Config>::RuntimeCall>;
+		/// Contains all calls that are allowed after the migration finished.
+		type RcCallEnabledAfterMigration: Contains<<Self as frame_system::Config>::RuntimeCall>;
```

Comment:
```suggestion
		type RcCallsEnabledDuringMigration: Contains<<Self as frame_system::Config>::RuntimeCall>;
		/// Contains all calls that are allowed after the migration finished.
		type RcCallsEnabledAfterMigration: Contains<<Self as frame_system::Config>::RuntimeCall>;
```

Or even something like `RcIntraMigrationCalls` and `RcPostMigrationCalls` for shorter naming.